src/createClassProxy.js

```diff
@@ -78,8 +75,9 @@ function proxyClass(Component) {
             'context',
             'constructor',
             'selector',
-            'props'
-          ]
+            'props',
+            'didUnmount'
+          ] // Can we get those automatically?
```

Comment:
I was in the middle of testing this thing. The idea is that we have to call some functions directly on the original component, while some other's on proxied one.